app/main.py

```diff
@@ -1,8 +1,44 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 in_class: int,
+                 in_mark: int,
+                 in_brand: str) -> None:
```

Comment:
Use the same names for the parameters and attributes. Also in other similar cases

Reply:
additional check: done. regarding same names for the parameters and attributes - could you pls tell me why? I use "in_arg" names in funcs for clear understanding which data came from the outside